/**:
  ros__parameters:
    detection_objects: false
    detection_points: true
    detection_margin_min: 1.0
    detection_margin_max: 3.0
    detection_area_front: 3.0
    detection_area_rear: 0.0
    detection_area_left: 2.0
    detection_area_right: 2.0
    detection_point_num_min: 100